src/blog/wasm-code-caching-for-devs.md

```diff
@@ -0,0 +1,123 @@
+﻿
```

Comment:
This needs the header metadata (like it had before)

Reply:
Add this:

```
---
title: 'Code caching for WebAssembly developers'
author: '[Bill Budge](https://twitter.com/billb), putting the Ca-ching! in caching'
avatars:
  - bill budge
date: 2019-06-13
tags:
  - WebAssembly
  - internals
---
```

Reply:
Done